introduction.md

```diff
@@ -1 +1,5 @@
 yo, this is sankage, i like Ruby
+
+* Git
+* Javascript
+* Ruby
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?